dfx.json

```diff
@@ -1,6 +1,13 @@
 {
 	"dfx": "0.20.0",
 	"canisters": {
+		"signer": {
+			"candid": "src/backend/backend.did",
+			"package": "backend",
```

Comment:
I assume it's correct that "the signer" points to "backend".

Reply:
Yes, it is!  For now it uses the exact same Wasm.